index.html

```diff
@@ -80,9 +81,10 @@ <h3>WeRemote</h3>
             <ul class="lin-parent">
               <li><a href="./index.html">Home</a></li>
               <li><a href="./pages/about.html">About</a></li>
-              <li>Programe</li>
-              <li>Join</li>
```

Comment:
- [ ] You did a great job 😃 at adding the mobile menu navigation, but I see that when opening the hamburger menu when on the home page and trying to click on the About page from the mobile menu it is not redirecting me to the About page it is redirecting me to a broken page, so please try to fix that to let the user when being able to try to navigate to the about page from the mobile menu, so: ⤵️

- I see that the path of the about page is wrong as you see here you wrote  ```./pages/about.html```, so to fix that just try to write ```./about.html``` and it will work for you 😊💯

> Check below to see the difference 😊⤵️
> If you can not see the image clearly just click on it to open a new tab in the browser😊

| Your design| 
| --- |
|  ![Recording 2023-04-20 at 22 05 29](https://user-images.githubusercontent.com/83360636/233476063-9d315a68-e076-4592-a680-bb5a2d4fdb6a.gif)   |